lib/src/tasks/saucelabs_tests/sauce_unit_test_driver.dart

```diff
@@ -157,6 +157,18 @@ class SauceUnitTestDriver {
         ..add(
             ProcessSignal.SIGINT.watch().take(1).listen((_) => _quitDriver()));
 
+      // If the browser window is not active, focus events won't be dispatched in Firefox as expected.
+      //
+      // So, before running any tests, we'll ensure the browser window is active by clicking on the page with WebDriver.
+      //
+      // 1. Load `about:blank` to ensure that clicking on the page does nothing.
+      // 2. Move the cursor over a valid element so that the WebDriver doesn't break itself trying to click on nothing.
+      // 3. Click.
+      await _driver.get('about:blank'); // [1]
+      await _driver.mouse.moveTo(
+          element: await _driver.findElement(const By.tagName('body'))); // [2]
```

Comment:
do you also need to add `By` to the show clause coming from `'package:webdriver/io.dart'`